MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -1275,15 +1258,15 @@ private static bool IsSubtitleEmbedSupported(MediaStream subtitleStream, Subtitl
                 {
                     return false;
                 }
-                if (ContainerProfile.ContainsContainer(normalizedContainers, "mpegts"))
+                else if (ContainerProfile.ContainsContainer(normalizedContainers, "mpegts"))
```

Comment:
can we combine these `if`-s into one list of containers that don't support embedded subtitles?